parts/windows/windowscsehelper.ps1

```diff
@@ -73,6 +73,8 @@ $global:LatestContainerdPackage = "v1.7.1-azure.1/binaries/containerd-v1.7.1-azu
 # The latest containerd version that contains stable ABI
 $global:LatestContainerdPackagefor23H2 = "v1.7.9-azure.1/binaries/containerd-v1.7.9-azure.1-windows-amd64.tar.gz"
 
+$global:EventsLoggingDir = "C:\WindowsAzure\Logs\Plugins\Microsoft.Compute.CustomScriptExtension\Events\"

```

Comment:
awesome to see this works! I'll be able to take advantage within the secure bootstrapping client :)

Reply:
would you be able to point towards some documentation/further info as to how you found this directory?

Reply:
I'm also looking for more documentation about it. :) I just refer to the directory of linux's implementation, then try to look for the similar directory and do some tests. I will share more documentation with you after I get deeper into it.